rates.md

```diff
@@ -36,23 +37,43 @@ module RATES
 
     syntax PotStep ::= "drip"
  // -------------------------
-    rule <k> Pot . drip => call Vat . suck VOW THIS ( CHI +Rat (((DSR ^Rat (TIME -Int RHO) *Rat CHI) -Rat CHI) ) ) ... </k>
+    rule <k> Pot . drip => call Vat . suck VOW THIS ( PIE *Rat CHI *Rat ( DSR ^Rat (TIME -Int RHO) -Rat 1 ) ) ... </k>
          <this> THIS </this>
          <currentTime> TIME </currentTime>
-         <pot-chi> CHI => CHI +Rat (((DSR ^Rat (TIME -Int RHO) *Rat CHI) -Rat CHI) ) </pot-chi>
+         <pot-chi> CHI => CHI *Rat DSR ^Rat (TIME -Int RHO) </pot-chi>
          <pot-rho> RHO => TIME </pot-rho>
          <pot-dsr> DSR </pot-dsr>
          <pot-vow> VOW </pot-vow>
+         <pot-pie> PIE </pot-pie>
       requires TIME >=Int RHO
+       andBool DSR >=Rat 1 // to ensure positive interest rate
```

Comment:
Use `NOW` instead of `TIME` as a variable name, for uniformity with the rest of the K code.

Also, where did this check `DSR >=Rat 1` come from? I don't think it's guaranteed that interest rates are always giving the user money.

Reply:
https://github.com/makerdao/dss/blob/master/src/pot.sol#L139
`sub` reverts when the first argument (`rmul(rpow(dsr, now - rho, ONE), chi)`) is less than the second (`chi`), which means that it reverts when `rpow(dsr, now - rho, ONE) < ONE`, i.e., `dsr ^ (now - rho) < 1`, which in turn means `dsr < 1` since `now >= rho`.

Reply:
If it is not the intended business logic, then the code is wrong.